op-node/rollup/derive/kromampt_upgrade_transactions.go

```diff
@@ -0,0 +1,292 @@
565b34801561018557600080fd5b5061016b7f000000000000000000000000000000000000000000000000000000000000000081565b3373ffffffffffffffffffffffffffffffffffffffff7f00000000000000000000000000000000000000000000000000000000000000001614610277576040517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152602560248201527f4665655661756c743a20746865206f6e6c7920726563697069656e742063616e60448201527f2063616c6c00000000000000000000000000000000000000000000000000000060648201526084015b60405180910390fd5b60006102816104b8565b604080516020810182526000815290517fe11013dd0000000000000000000000000000000000000000000000000000000081529192507342000000000000000000000000000000000000099163e11013dd918491610308917f0000000000000000000000000000000000000000000000000000000000000000916188b891906004016106c3565b6000604051808303818588803b15801561032157600080fd5b505af1158015610335573d6000803e3d6000fd5b505050505050565b3373ffffffffffffffffffffffffffffffffffffffff7f00000000000000000000000000000000000000000000000000000000000000001614610402576040517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152602560248201527f4665655661756c743a20746865206f6e6c7920726563697069656e742063616e60448201527f2063616c6c000000000000000000000000000000000000000000000000000000606482015260840161026e565b600061040c6104b8565b9050600061044b7f00000000000000000000000000000000000000000000000000000000000000005a8460405180602001604052806000815250610624565b9050806104b4576040517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152601d60248201527f4665655661756c743a20455448207472616e73666572206661696c6564000000604482015260640161026e565b5050565b60007f0000000000000000000000000000000000000000000000000000000000000000471015610590576040517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152604a60248201527f4665655661756c743a207769746864726177616c20616d6f756e74206d75737460448201527f2062652067726561746572207468616e206d696e696d756d207769746864726160648201527f77616c20616d6f756e7400000000000000000000000000000000000000000000608482015260a40161026e565b6000479050806000808282546105a69190610707565b9091555050604080518281527f000000000000000000000000000000000000000000000000000000000000000073ffffffffffffffffffffffffffffffffffffffff166020820152338183015290517fc8a211cc64b6ed1b50595a9fcb1932b6d1e5a6e8ef15b60e5b1f988ea9086bba9181900360600190a1919050565b600080600080845160208601878a8af19695505050505050565b6000815180845260005b8181101561066457602081850181015186830182015201610648565b81811115610676576000602083870101525b50601f017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0169290920160200192915050565b6020815260006106bc602083018461063e565b9392505050565b73ffffffffffffffffffffffffffffffffffffffff8416815263ffffffff831660208201526060604082015260006106fe606083018461063e565b95945050505050565b60008219821115610741577f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b50019056fea164736f6c634300080f000a")
```

Comment:
Since a user who derives kroma through KromaMPT hardfork in the far future, not using snapshot, would have a trouble to successfully derive the chain when we use the bytecode in bindings which can be changed dynamically. So we hardcoded the bytecode in current bindings like a snapshot.

Reply:
1. We've had this discussion before, and it took the form of embedding bytecode. https://github.com/kroma-network/kroma/pull/398#discussion_r1881229723
2. It would be nice to be able to verify via a hash or something, but does the hash value for the deployment byte code exist as an output here?

Reply:
Thanks for explanation.  I just showed you the logic. Let's keep it as is